media-libs/shaderc/shaderc-2017.2.ebuild

```diff
@@ -0,0 +1,81 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+COMMIT_ID="7a23a01742b88329fb2260eda007172135ba25d4"
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
+
+inherit cmake-multilib python-any-r1 vcs-snapshot
```

Comment:
Don't use vcs-snapshot, the dirname is predictable here. Define `S` instead.

Reply:
I don't like fiddling with S. Why not use a suitable eclass for this?

Reply:
This is not 'suitable eclass for this'. If you read the documentation on top of the eclass you'd learn that it's meant for *unpredictable* contents such as old GitHub tarballs. The eclass would've been deprecated if not the fact that occasionally we hit such tarballs (like 0.05%).

Reply:
Ok.